packages/davinci-client/src/lib/collector.types.ts

```diff
@@ -45,7 +45,7 @@ export interface ActionCollector {
   };
 }
 
-export interface FlowCollector extends ActionCollector {}
-export interface SocialLoginCollector extends ActionCollector {}
-export interface PasswordCollector extends SingleValueCollector {}
-export interface TextCollector extends SingleValueCollector {}
+export type FlowCollector = ActionCollector;
+export type SocialLoginCollector = ActionCollector;
```

Comment:
Are these aliases going to have a more significant difference going forward? Like SocialLoginCollector and FlowCollector, will they have differences in the type? Or are we going to always have these aliases for the same type?

Reply:
I'm expecting these types to evolve differently as they mature. Right now, they just happen to be the same. Social Login isn't fully developed yet and will likely have additional configuration for styling and such.